### DISCUSSION TITLE
Dafoam y+ accuracy

### DISCUSSION THREAD

**Discussion by byeonghoon-ahn** [mdolab/dafoam#170] [General]
Hello，
I see that the example of dafoam is set to y+>1. When I calculate wing using CFD++ or other CFD solvers, the mesh is set to y+< 1. 
Since I haven't learned openfoam before, Does the calculation of Y+> 1 meet the experimental results?
Kindly help me in this regard.

**Comment by friedenhe** [comment]
DAFoam uses the OpenFOAM's built-in y+ calculation function. Can you run OpenFOAM's built-in y+ calculation utility and see if it agrees with other CFD solvers?

**Reply by byeonghoon-ahn** [reply]
Thank you very much for the reply~

**Reply by 15175829310** [reply]
Why is it that the y+ calculated by openfoam is ≈30, but the y+ calculated by DArhoSimplecFoam is ≈0.016? I don't know what the problem is. I look forward to your reply.

**Comment by 15175829310** [comment]
Why is it that the y+ calculated by openfoam is ≈30, but the y+ calculated by DArhoSimplecFoam is ≈0.016? I don't know what the problem is. I look forward to your reply.

**Comment by friedenhe** [comment]
How did you calculate the yPlus? Some screenshots will be useful.
